docs/source/getting-started/index.rst

```diff
@@ -34,70 +34,70 @@ Interested in navigating directly to the content that you're curious about? Sele
                     <div class="header">
                         Setting up MLflow Servers
                     </div>
+                    <p>
+                        Learn how to start a MLflow Tracking Server and the MLflow UI Server locally
+                    </p>
                 </a>
-                <p>
-                    Learn how to start a MLflow Tracking Server and the MLflow UI Server locally
-                </p>
             </div>
             <div class="simple-card">
                 <a href="logging-first-model/step2-mlflow-client.html" >
                     <div class="header">
                         Using the MLflow Client
                     </div>
+                    <p>
+                        Connect to the Tracking Server with the MLflow Client and learn to search for experiments
+                    </p>
                 </a>
-                <p>
-                    Connect to the Tracking Server with the MLflow Client and learn to search for experiments
-                </p>
             </div>
             <div class="simple-card">
                 <a href="logging-first-model/step3-create-experiment.html" >
                     <div class="header">
                         Create your first Experiment
                     </div>
+                    <p>
+                        Explore the MLflow UI and create your first MLflow experiment with a unique name and identifying tags
+                    </p>
                 </a>
-                <p>
-                    Explore the MLflow UI and create your first MLflow experiment with a unique name and identifying tags
-                </p>
             </div>
             <div class="simple-card">
                 <a href="logging-first-model/step4-experiment-search.html" >
                     <div class="header">
                         Search Experiments by tags
                     </div>
+                    <p>
+                        Learn the benefits of relevant identifying tags as you execute a search for experiments containing identifying tag values
```

Comment:
this looks a bit complex, shall we just say "Learn how to apply tags to MLflow experiments and search experiments by tags"?